### DISCUSSION TITLE
GPU temperature target option not available for the Zephyrus G15 2021.

### DISCUSSION THREAD

**Discussion by NJ1mi** [seerge/g-helper#1792] [Q&A]
Hello,
I'm using G-Helper on the Zephyrus G15 and I'm having trouble finding the GPU temperature target option. Is there any way to fix this? I've attached a screenshot of it. Thanks
![Screenshot (2)](https://github.com/seerge/g-helper/assets/154861388/43bb1b73-c902-44b2-87e7-00a6c70192bb)

**Comment by RAGE-HERO7** [comment]
Bro it's not supported by your hardware
I guess your Zephyrus is a 2021model

Yesterday I asked almost the same question and got the answer for it
Here is the link to that post

https://github.com/seerge/g-helper/discussions/1785
